storage/src/main/java/com/tencent/rss/storage/handler/impl/LocalFileClientReadHandler.java

```diff
@@ -53,29 +60,88 @@ public LocalFileClientReadHandler(
     this.shuffleServerClients = shuffleServerClients;
   }
 
-  @Override
-  public ShuffleDataResult readShuffleData(int segmentIndex) {
+  public ShuffleIndexResult readShuffleIndex() {
+    boolean readSuccessful = false;
+    ShuffleIndexResult shuffleIndexResult = null;
+    RssGetShuffleIndexRequest request = new RssGetShuffleIndexRequest(
+        appId, shuffleId, partitionId, partitionNumPerRange, partitionNum);
+
+    for (ShuffleServerClient shuffleServerClient : shuffleServerClients) {
```

Comment:
For replica = 2, read index from server1 and it's possible to read data from server2?

Reply:
No, this is actually an one item list, no replica support in this pr, this is only following the invention of current way to read data(index or shuffle data). I will make chagne to this part in the following pr.